config/routes.rb

```diff
@@ -1,31 +1,41 @@
 Rails.application.routes.draw do
   root "home#index"
+  get "/about", to: "home#about"
 
-  get "/auth/:provider/callback", to: "sessions#create"
-  get "/auth/failure", to: "sessions#failure"
-  post "/login", to: "sessions#create"
-  delete "/logout", to: "sessions#destroy"
-  get "/admin/dashboard", to: "admin/dashboard#index"
-  post "/business/approval", to: "businesses#approve"
-
+  get    "/auth/:provider/callback", to: "sessions#create"
+  post   "/login",                   to: "sessions#create"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.